test/metabase/query_processor/middleware/annotate_test.clj

```diff
@@ -508,6 +511,22 @@
       (is (= {:base_type :type/Float}
              (infered-col-type [:case [[[:> [:field (mt/id :venues :price) nil] 2] [:+ [:field (mt/id :venues :price) nil] 1]]]]))))))
 
+(deftest ^:parallel datetime-arithmetics?-test
+  (is (#'annotate/datetime-arithmetics? [:+ [:field-id 13] [:interval -1 :month]]))
```

Comment:
field-id is a legacy MBQL syntax, the new syntax is now `[:field 13 nil]`.

See this [PR](https://github.com/metabase/metabase/pull/14897).

And also I think we shouldn't hardcode the id, you should use sth like `(mt/id :checkin :dates)`, this will return a id for the `checkin.dates`. 

One benefit of using `mt/id` is that it returns based on the database that we bound to, I think the default db is `test-data`, but you can change it using `mt/dataset`.
This is handy when you test against different drivers, which will switch between DBs.

Reply:
Updated https://github.com/metabase/metabase/pull/26279/commits/aa30353a164a6febb6c932ba65208cf023d62eba